spring-cloud-data-core/src/main/java/org/springframework/cloud/data/core/ModuleCoordinates.java

```diff
@@ -39,11 +38,11 @@
 
 	private String version;
 
-	protected ModuleDefinition() {
+	protected ModuleCoordinates() {
 		// For (subclass) JSON deserialization only
 	}
 
-	protected ModuleDefinition(String name, String artifactId, String groupId, String version) {
+	protected ModuleCoordinates(String name, String artifactId, String groupId, String version) {
```

Comment:
Good question - I think you introduced `ModuleDefinition` into this project. ;) It does look like this should be `public`. I'll modify it.